.github/workflows/trigger-e2e-tests.yml

```diff
@@ -62,14 +62,14 @@ jobs:
 
   trigger-e2e-tests:
     needs: [ tag ]
-    runs-on: [ self-hosted, gen3, small ]
+    runs-on: ubuntu-latest
```

Comment:
Can we still pin the version?  usually unpinned container versions result in something breaking mysteriously like 1yr later ;-)

Reply:
We have a bunch of using `ubuntu-latest` in the code, I'll create a separate issue for this.